src/components/02_atoms/FileUpload/FileUpload.js

```diff
@@ -10,19 +10,13 @@ import api from './../../../utils/api/api';
 const Element = styled('div')`
   border: 2px dashed grey;
   border-radius: 3px;
-  padding: 80px;
-  width: 100%;
-  position: relative;
+  display: flex;
```

Comment:
Does someone know the state of flexbox IE 11 support?

Reply:
https://caniuse.com/#feat=flexbox

Under the known issues tab - there is a good definition of "partial" support.